### ISSUE TITLE
[W11][DX11][5700 XT] God of War - checkerboarded display rendering - RDNA1

### ISSUE THREAD

**Issue by RareMv** [doitsujin/dxvk#2443]
Tried God of War patched to 1.01 on Windows 11, 5700 XT, drivers Adrenaline 22.1.1 and NimeZ 22.1.1 using DXVK 1.9.3 and DXVK 19.3 Async. 

The image looks like a chess board. Like it's being checkerboarded, where 50% of the available pixels are rendering fine, the other 50% are a transparent square or rendered in delay. 

### Software information
God of War (2022 PC release) / version 1.01

### System information
- GPU: RX 5700 XT
- Driver: Adrenalin 22.1.1
- Wine version: Windows 11
- DXVK version: 1.9.3

### Apitrace file(s)
- I don't have one

### Log files
- [d3d11.log:](https://mega.nz/file/vckCWLrR#OX4rLudCPnuGYrCjcIK55Nqin55wi6HFlB5I4vf5tf4)
- [dxgi.log](https://mega.nz/file/KdsynTQb#HWphk1b19lNHEGvjs4V4NKjIcE2e8FZF0cIpr8MeCPY)

How the game is displayed: 
<a href="https://ibb.co/3T3b2ht"><img src="https://i.ibb.co/QJgVLms/Go-W-2022-01-17-14-47-48-770.png" alt="Go-W-2022-01-17-14-47-48-770" border="0"></a>

**Comment by RareMv** [comment]
Reading online, I've saw people with RDNA2 and even Maxwell v1 (GTX 750 Ti) on Windows and even DXVK run on Linux having perfect rendering with the game. So I can only assume with a bug with RDNA1 and it's Vulkan implementation or the way DXVK is trying to mimic DX 11_1 feature level on RDNA1 where something craps out.

**Comment by doitsujin** [comment]
What DXVK does on RDNA1 is identical to what it does on RDNA2 so I strongly suspect a driver bug.

**Comment by RareMv** [comment]
Neither of the DX11 performance tweaks affects how the image is rendered. I do think it's a driver bug :(

**Comment by ryzen23** [comment]
i have the same issue on my RX 5600XT [https://imgur.com/erlUKMf](ur)

**Comment by Ryusennin** [comment]
For reference, there is no such checkerboard bug on Win10 with the same Radeon 22.1.1 driver. However DXVK performance is noticeably inferior to native DX11.

**Comment by doitsujin** [comment]
Note that performance issues are primarily caused by the game's heavy reliance on NVAPI and AMDAGS which do not work together with DXVK. The current `master` branch has a workaround which mimics what the game does with those external libraries.

**Comment by RareMv** [comment]
Brought up the issue with AMD on Reddit, two reps acknowledged it and said the Vulkan team was able to reproduce the issue internally and are working on it.

@Ryusennin Good to know, thanks for the details. You also have RDNA1?

**Comment by SirGouki** [comment]
> For reference, there is no such checkerboard bug on Win10 with the same Radeon 22.1.1 driver. However DXVK performance is noticeably inferior to native DX11.

I stumbled in here from a recommendation to use this for PSO2, and I always check issues of stuff on github before I test it myself.  I want to point out that the drivers for Linux are almost **never** the same as they are for Windows.  Also, for linux their are 2 sets of drivers, the regular official drivers and the good official drivers.  I believe the 2nd set is called MESA.  The reason why these are usually the best option, is because the regular official drivers are almost always buggy (even on windows unfortunately, because on windows we don't get the MESA drivers).  I have no idea why this is a thing, and why the MESA drivers aren't the default option given how long they've been the best.  It should be fairly obvious why the drivers on linux are not ever going to be the same as they are on windows (in case its not: different window managers (not 100% sure this is still an issue, I know it was part of it at one point), different kernal level functionality, different file format for system objects (.so) vs dynamic link libraries (.dll), different available graphics library exposures (opengl and vulkan, vs opengl, vulkan and DirectX note that WINE works by wrapping DX calls to opengl, which is why you don't need seperate graphics drivers for WINE), I'm sure there are other reason why this is still the case)

In short: it is fallacy to expect the same numbered version of drivers for anything to be identicle across wholly different operating systems, and this can not be used to rule out the drivers themselves as a problem.

**Comment by Ryusennin** [comment]
@RareMv Sorry, no RDNA. I should have mentioned that I did my test on a Vega (GCN5).

**Comment by noctngu** [comment]
Is the checkerboarding issue still present in the latest DXVK build?

**Comment by K0bin** [comment]
It's not a DXVK issue, so it depends on whether AMD has fixed their driver.

**Comment by boda290** [comment]
Hello.. is that problem been fixed after amd latest update .. adrenaline edtion ?

**Comment by RareMv** [comment]
As of 22.2.3 it wasn't fixed. Can't speak for 22.3.1. You're better off asking on Reddit or Guru3D honestly.

**Comment by RareMv** [comment]
While I haven't retested myself, Reddit comments mentioned this has been fixed as of ... 22.4.2 I think.

**Comment by Blisto91** [comment]
@RareMv Would you be able to test with that driver? 🙂

**Comment by K0bin** [comment]
I'll just go ahead and close this. Hopefully the random redditor is right but we don't work around bugs in the AMD Windows Vulkan driver anyway, so there's nothing for us to do here either way.
